src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/TreeNodeGenerator.tt

```diff
@@ -179,6 +179,12 @@ namespace Microsoft.SqlTools.ServiceLayer.ObjectExplorer.SmoModel
 				{
 					WriteLine("            var child = new SmoTreeNode();");
 					WriteLine("            child.IsAlwaysLeaf = true;");
+					if(type.EndsWith("s")) 
```

Comment:
are we sure that none of the types well need to support will end with 's'?  Also, it looks like we're missing a space between f and ( in the "if (" statement.